Package.swift

```diff
@@ -22,9 +22,9 @@ var package = Package(
     ],
     dependencies: [
         .package(url: "https://github.com/IBDecodable/IBDecodable.git", from: "0.5.0"),
-        .package(url: "https://github.com/Carthage/Commandant.git", .upToNextMinor(from: "0.17.0")),
         .package(url: "https://github.com/jpsim/SourceKitten.git", from: "0.29.0"),
         .package(url: "https://github.com/phimage/XcodeProjKit.git", from: "2.2.0"),
+        .package(url: "https://github.com/apple/swift-argument-parser", from: "0.5.0"),
```

Comment:
Why don't we use version 1.0.0? If you want to use an older version, could you leave a comment to explain the reason.

Reply:
It was initially meant for keeping the compatibility with Swift 5.1. But now that we can remove that from CI, we can use 1.0.0 👍

Fixed at 1f92f5a

Reply:
This might also be a good opportunity to consider bumping up `swift-tools-version` for this package since testing only supports 5.2 or higher. What do you think @kateinoigakukun

Reply:
Sounds good :)

Reply:
Fixed at f62d989 and 0378b70. I also made changes needed to fix the build errors. I believe they are not significant changes but I would appreciate it if you could take a look.